src/components/Header.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState, Suspense } from "react";
+import Search from './blocks/Search';
+import Logo from '../components/blocks/Logo';
+const Form = React.lazy(() => import('../components/blocks/Form'));
+
+const movieEmpty = {
+    title: '',
+    year: '',
+    genres: [],
+    releaseDate: ''
+}
+
+const Header: React.FC = () => {
+    const [showModal, setShowModal] = useState(false);
+
+    const modal = showModal 
+                        && (
+                        <Suspense fallback={<div>Загрузка...</div>}>
```

Comment:
If you are use english, it's better to rename to `Loading`

Reply:
fixed it